AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -12,16 +12,33 @@ import UIKit
 
 class LiveTranscriptionOptionsViewController: UIViewController, UITextFieldDelegate {
     var model: MeetingModel?
+    var storyBoard: UIStoryboard?
+    var languageDetectionViewController: AutomaticLanguageDetectionViewController?
```

Comment:
Same comments as above. Avoid having names like `LanguageDetection` or `Detection` in general. 

Naming in general should be based on parameter in consideration, not the functionality it serves.

Reply:
will be updated in next commit.